auditlog/api.go

```diff
@@ -205,3 +60,47 @@ func get(path string) (*http.Response, error) {
 
 	return resp, nil
 }
+
+// EventResponse represents an Audit Log event stored in your WorkOS Audit Log.
+type EventResponse struct {
+	Event
+
+	ID              string `json:"id"`
+	Object          string `json:"object"`
+	LocationCity    string `json:"location_city"`
+	LocationState   string `json:"location_state"`
+	LocationCountry string `json:"location_country"`
+	AppID           string `json:"app_id"`
+}
+
+// ListRequestParams allows you to confire FindAll, or List request to paginate
+// any entries after & before a given time or by a specific action.
+type ListRequestParams struct {
+	StartingAfter string
+	EndingBefore  string
+	Limit         int
+}
+
+func (p ListRequestParams) GetStartingAfter() string {
+	if p.EndingBefore != "" && p.StartingAfter != "" {
+		return ""
+	}
+	return p.StartingAfter
+}
+
+func (p ListRequestParams) GetEndingBefore() string {
+	return p.EndingBefore
+}
+
+func (p ListRequestParams) GetLimit() int {
+	if p.Limit <= 0 {
+		return 10
+	}
+	return p.Limit
+}
+
+type ListMeta struct {
+	HasMore    bool   `json:"has_more"`
+	TotalCount uint32 `json:"total_count"`
+	URL        string `json:"url"`
```

Comment:
Could be helpful to document what these fields may contain. What's URL in this example?

Reply:
`URL` is the endpoint which the data belongs to. This is useful for nested lists where one may want to paginate the child's list. For example..
```json
GET /foo
{
    "object": "foo",
    "url": "/foo",
    "has_more": true,
    "data": [{
        "object": "foo",
        "id": "foo_01DAYKK3T3YCX7F2FM9J4ZCRB1",
        "bars": {
            "object": "bar",
            "url": "/foo/foo_01DAYKK3T3YCX7F2FM9J4ZCRB1/bar",
            "has_more": false,
            "data": [{
                "object": "bar",
                "id": "bar_01DAYKK3T3YCX7F2FM9J4ZCRB1",
                "value": "foobar"
            }]
        }
    }]
}
```